### ISSUE TITLE
Suggestions, comments & general questions

### ISSUE THREAD

**Issue by benmarwick** [benmarwick/onboarding-reproducible-compendia#1]

**Comment by jhollist** [comment]
So this was an exciting find on a Saturday nigh!

Quick plea...  Requiring MIT for code is going to be problematic for many feds.  Clear guidance on licensing of code developed by federal employees is essentially non-existent.  As such the use of true open source licenses (e.g. OSI approved) is not yet allowed.   I've been pushing EPA for an answer on this issue for a very long time (going on multiple years since my first inquiry).

Short version, could you also allow CC0 for code.  We do have permission to use that.

**Comment by cboettig** [comment]
:+1:

**Comment by benmarwick** [comment]
Thanks @jhollist, this is great feedback. I've made an edit to https://github.com/benmarwick/onboarding-reproducible-compendia/blob/master/packaging_guide.md in response to your plea. Do let us know if you see anything else that is at odds with how your making your work reproducible.

**Comment by jhollist** [comment]
@benmarwick and @cboettig Thanks!

The change looks good.  I also dug more into the linked document from Victoria Stodden.  She addresses CCO explicitly and counts it as an appropriate license for Repredocubile Research compendia.  I do think it is also a good idea to have explicitly listed in this repo as well since other efforts (i.e. JOSS) are built off of the OSI licenses and don't allow CC0 (for good reasons, btw).

And I will be watching this with interest.  If you are looking for more help, count me in!

**Comment by mustafaascha** [comment]
First, let me mention that I'm really happy to participate in this discussion!

1. While I completely agree that a standardized project folder structure makes things easier for users who want to browser a compendium, the packaging guide seems to reflect the package submission guidelines for ROpenSci rather than packaging compendia. 

In some of my own work (e.g. [here](https://github.com/mustafaascha/brain-mets-seer)), I use multiple custom packages. This is because they serve different purposes, where one package does the extraction/transformation and another package does the analysis. The package that I expect to be useful to others (extraction/transformation of cancer registry data) is better-documented, whereas the package specific to my project is not as well-documented. 

Given the use of several packages, what would the recommendation be for a project filetree? 

2. I work primarily with human subjects research data, so data sharing isn't quite in the cards. Are there any expectations/guidelines for pseudo-data sharing in this case? For example, a synthetic dataset to enable package testing, or DDL to make explicit the data format.
